lib/phoenix_live_view/html_tokenizer.ex

```diff
@@ -17,6 +19,33 @@ defmodule Phoenix.LiveView.HTMLTokenizer do
 
       "#{location} #{exception.description}"
     end
+
+    def code_snippet(source, meta, arrow_padding) do
```

Comment:
Adding this function here since `ParseError` is used for both Engine and Tokenizer. Wdyt?

Reply:
:+1: